lib/rdf/serialization.ts

```diff
@@ -40,298 +41,305 @@ const xsdUnsignedShort = xsd + 'unsignedShort';
 const xsdUnsignedByte = xsd + 'unsignedByte';
 const xsdPositiveInteger = xsd + 'positiveInteger';
 
-export const exportLiteralTerm = (term: string, dataFactory: DataFactory): Literal => {
-  const [, encoding, datatype, language, value] = term.split('\u0001');
-  switch (datatype) {
-    case rdfLangString:
-      if (language !== '') {
-        return dataFactory.literal(value, language);
-      }
-      return dataFactory.literal(value);
-    default:
-      return dataFactory.literal(value, dataFactory.namedNode(datatype));
+export class RdfSerialization {
+  constructor(readonly prefixes: TSRdfPrefixes) {
   }
-}
 
-export const importLiteralTerm = (term: Literal, rangeBoundary = false): string => {
-  const { language, datatype, value } = term;
-  if (language !== '') {
-    return `\u0001\u0001${rdfLangString}\u0001${language}\u0001${value}`;
-  }
-  if (!datatype || datatype.value === xsdString) {
-    return `\u0001\u0001${xsdString}\u0001\u0001${value}`;
-  }
-  switch (datatype.value) {
-    case xsdInteger:
-    case xsdDouble:
-    case xsdDecimal:
-    case xsdNonPositiveInteger:
-    case xsdNegativeInteger:
-    case xsdLong:
-    case xsdInt:
-    case xsdShort:
-    case xsdByte:
-    case xsdNonNegativeInteger:
-    case xsdUnsignedLong:
-    case xsdUnsignedInt:
-    case xsdUnsignedShort:
-    case xsdUnsignedByte:
-    case xsdPositiveInteger:
-      if (rangeBoundary) {
-        return `\u0001number:${fpstringEncode(value)}\u0001`;
-      }
-      return `\u0001number:${fpstringEncode(value)}\u0001${datatype.value}\u0001\u0001${value}\u0001`;
-    case xsdDateTime:
-      const timestamp = new Date(value).valueOf();
-      if (rangeBoundary) {
-        return `\u0001datetime:${fpstringEncode(timestamp)}`;
-      }
-      return `\u0001datetime:${fpstringEncode(timestamp)}\u0001${datatype.value}\u0001\u0001${value}\u0001`;
-    default:
-      return `\u0001\u0001${datatype.value}\u0001\u0001${value}\u0001`;
+  exportLiteralTerm = (term: string, dataFactory: DataFactory): Literal => {
```

Comment:
Could you refactor to use ES6's method declaration syntax rather than arrow function assignments?